gulp-core-build-sass/src/SassTask.ts

```diff
@@ -186,7 +192,10 @@ export class SassTask extends GulpTask<ISassTaskConfig> {
               const value: string = classNames[key];
               let line: string = '';
               if (key.indexOf('-') !== -1) {
-                this.logWarning(`The local CSS class '${key}' is not camelCase and will not be type-safe.`);
+                let message: string = `The local CSS class '${key}' is not camelCase and will not be type-safe.`;
+                this.taskConfig.warnOnCssTypeSafety ?
+                  this.logVerbose(message) :
```

Comment:
warnOnCssTypeSafety === true
send verbose message
warnOnCssTypeSafety  === false
send warning (this will result exit code = 1)